baseplate/lib/edge_context.py

```diff
@@ -61,6 +63,10 @@ def validate(self, token: bytes) -> "AuthenticationToken":
 
         return InvalidAuthenticationToken()
 
+    @functools.lru_cache()
+    def _prepare_key(self, key_data: bytes) -> "jwt.algorithms.RSAPublicKey":
```

Comment:
this fails mypy because the import of `jwt.algorithms.RSAPublicKey` is internally conditional on `cryptography` isn't installed and it looks like mypy doesn't run that way?

Reply:
Ah weird. Yeah, mypy must not know what to do with that conditional. The linters should have cryptography installed.